source/integrate/mattermost-mission-collaboration-for-m365.rst

```diff
@@ -0,0 +1,155 @@
+Mattermost Mission Collaboration for Microsoft 365 (Beta)
+=========================================================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+Mattermost Mission Collaboration for Microsoft extends Microsoft Office365, Microsoft Outlook, and Microsoft Teams for mission-critical coordination, command and control, incident response, and DevSecOps workflows in demanding environments, including air-gapped and classified networks.
+
+.. image:: ../images/External-Collaboration-with-Enterprise-Control-Microsoft.png
+  :alt: Mattermost Mission Collaboration for Microsoft application extends Enterprise communications with secure external collaboration controlled by IT by accessing Mattermost from your Microsoft Office365, Teams, and Outlook user experiences.
+
+This app is designed to work with Microsoft 365, Outlook, and Teams.
+
+Setup
+-----
+
+Create an MS Teams app in Azure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Sign in to the `Azure portal <https://portal.azure.com/>`_ using an admin Azure account.
+
+2. Go to your **Azure Portal > Microsoft Entra ID**.
+
+3. Go to **App registrations**.
+
+4. Create a new app registration by selecting **Add > App registration**:
+
+  - Give it a name
+  - Accounts in this organizational directory only (single tenant)
+  - No redirect URIs
+
+5. Go to your newly created application and copy the **Application (client) ID** and **Directory (tenant) ID** values. You'll need those later to configure the plugin.
+
+  .. image:: ../images/remember-tenant-client.png
+    :alt: Go to your newly created application and copy the Application (client) ID and Directory (tenant) ID values. You'll need those later to configure the plugin.
+
+6. Go to **Certificates and secrets** to generate a new client secret. Make a copy of the secrfet value, as it will only be shown once. You'll need this value to configure the plugin.
```

Comment:
```suggestion
6. Go to **Certificates and secrets** to generate a new client secret. Make a copy of the secret value, as it will only be shown once. You'll need this value to configure the plugin.
```